@kiva/kv-activity-feed/src/index.ts

```diff
@@ -108,4 +108,42 @@ export default class ActivityFeedService {
 
 		return success;
 	}
+
+	/**
+	 * Adds a like comment for the current user for the provided activity ID
+	 *
+	 * @param commentId The comment ID to add a like
+	 * @returns Whether the like was added successfully
+	 */
+	async addLikeToComment(commentId: string): Promise<boolean> {
```

Comment:
Please add unit tests for these new methods, following the pattern of the other methods in this file.

Reply:
For this we need to update the `addComment` to return a valid commentId and add a child reaction then.
Said that, we also need to update the current flow we already have in CPS. Makes sense?

Reply:
The unit test just needs to verify that we are using the correct 3rd party method, a real quick addition, like my most recent PR.

Reply:
Gotcha, thanks for checking.